dist/selleckt.js

```diff
@@ -419,12 +419,12 @@ _.extend(SellecktPopup.prototype, {
 
     refreshItems: function(items){
         var $rendered = _.map(items, function(item){
-            var itemEl = Mustache.render(this.itemTemplate, {
+            var itemEl = Mustache.render(this.itemTemplate, _.extend({
                 itemTextClass: this.itemTextClass,
                 itemClass: this.itemClass,
                 label: item.label,
```

Comment:
Thanks @codazzo, fixed in https://github.com/BrandwatchLtd/selleckt/pull/77/files#diff-21d2211cbd23a29ee34ef26c5d128e54R121.